### PR TITLE
Initial definitions for the cd-excision project

### PR DESCRIPTION
**Pull request by EgbertRijke**
This PR sets up some initial definitions for #930.

### CODE REVIEW

File: src/orthogonal-factorization-systems/cd-structures.lagda.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    (Σ : cd-structure) →
+    {l1 l2 l3 l4 : Level} {A : UU l1} {B : UU l2} {X : UU l3} {Y : UU l4} →
+    (f : A → B) (g : X → Y) → hom-arrow f g → UU (α l1 l2 l3 l4)
+  is-distinguished-square-cd-structure Σ f g σ =
+    type-Prop (Σ f g σ)
+```
+
+## Exernal links
+
+- [cd-structure](https://ncatlab.org/nlab/show/cd-structure) at the $n$lab
```
Comment (fredrik-bakke) [inline src/orthogonal-factorization-systems/cd-structures.lagda.md:73]:
```suggestion
- [cd-structure](https://ncatlab.org/nlab/show/cd-structure) at the $n$Lab
```

File: src/orthogonal-factorization-systems/regular-cd-structures.lagda.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Regular cd-structures
+
+```agda
+module orthogonal-factorization-systems.regular-cd-structures where
+```
```
[the remark this answers is not in the document]
Comment (fredrik-bakke) [reply src/orthogonal-factorization-systems/regular-cd-structures.lagda.md:5]: I'd suggest just omitting this file for this PR.

File: src/orthogonal-factorization-systems/regular-cd-structures.lagda.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+</details>
+
+## Idea
+
+A {{#concept "regular cd-structure"}} is a
+[cd-structure](orthogonal-factorization-systems.cd-structures.md) which
+satisfies the following three axioms:
+
+1. Every distinguished square is [cartesian](foundation.pullback-squares.md).
+2. The codomain of every distinghuished square is an
```
Comment (fredrik-bakke) [inline src/orthogonal-factorization-systems/regular-cd-structures.lagda.md:22]:
```suggestion
2. The codomain of every distinguished square is an
```

File: src/synthetic-homotopy-theory/cocartesian-morphisms-arrows.lagda.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+## Definitions
+
+### The predicate of being a cocartesian morphism of arrows
+
+```agda
+module _
+  {l1 l2 l3 l4 : Level} {A : UU l1} {B : UU l2} {X : UU l3} {Y : UU l4}
+  (f : A → B) (g : X → Y) (h : hom-arrow f g)
+  where
+```
```
[the remark this answers is not in the document]
Comment (EgbertRijke) [reply src/synthetic-homotopy-theory/cocartesian-morphisms-arrows.lagda.md:50]: I added the formalization now

**Pull request merged**